### ISSUE TITLE
Is it possible to define a bool in the config file?

### ISSUE THREAD

**Issue by raspberry-jenshen** [TatsuUkraine/dart_environment_config#21]
Hey, awesome package!

Have a question:

Is it possible to define a bool in the config file?

```
environment_config:
  path: environment_config.dart

  fields:
    fake_api:
      type: bool
      const: TRUE
      default: false
      config_field: TRUE
      static: TRUE
```

Currently, it crashes during config parsing with an error 

`type 'bool' is not a subtype of type 'String'`

Thanks!

version: 2.2.2

**Comment by TatsuUkraine** [comment]
Hey) thanks.

Hm, I will test it in a few min and will let you know)

**Comment by TatsuUkraine** [comment]
@JenshenSoft so parser expects that all values that he gets are Strings. So to fix it you simply need to define `default` as a string, like `default: "false"`

I don't know if it's a bug) but I can fix it if you want

**Comment by TatsuUkraine** [comment]
@JenshenSoft if something, I just published new version with fix for boolean default values (2.2.3).

Also, I don't know for what project you're using my package, but if it's Flutter (and you using version 1.17) it's probably better to use compile-time variables to define environment-specific configurations - you can read more about it [here](https://medium.com/@tatsu.ukraine/flutter-1-17-no-more-flavors-no-more-ios-schemas-command-argument-that-solves-everything-8b145ed4285d)

**Comment by raspberry-jenshen** [comment]
Hi,

Thank you a lot!

Will take a look!
